sw_utils/decorators.py

```diff
@@ -50,6 +50,10 @@ def default_log_before(retry_state: 'RetryCallState') -> None:
     default_logger.log(logging.INFO, msg, *args)
 
 
+def default_after(future: Any) -> None:
+    raise future.outcome.exception()
```

Comment:
Pls add comment that you re-raise original exception and/or suppress 'RetryError' messages